tests/gem5/suite_tests/README.md

```diff
@@ -0,0 +1,7 @@
+# Suite Tests
+
+These tests runs a workload from every suite in gem5 resources.
```

Comment:
Does it run the workload to completion or just make sure the workload can be "loaded"?

Reply:
This runs it to make sure the workload is loaded, it runs to 10 billion ticks.

Reply:
You should update the description with that information 😄